### PR TITLE
135 adding hrmp channel establishment article

### LINKED ISSUE #135: Article on HRMP establishment

### PR DESCRIPTION
**Pull request by bgodlin**

### CODE REVIEW

**Review by N1ghtStorm — APPROVED** [review]

File: src/establishing-hrmp-channel.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Establish HRMP channel
+
+To establish communication between your parachain and SORA, you'll need to create HRMP channels that act as bridges for messages. This guide will walk you through the process, using information from the official [Substrate documentation](https://docs.substrate.io/reference/how-to-guides/parachains/add-hrmp-channels/) but applied for SORA.
```
Comment (WRRicht3r) [inline src/establishing-hrmp-channel.md]:
```suggestion
To establish communication between your parachain and SORA, you must create HRMP channels that act as bridges for messages. This guide will walk you through the process using information from the official [Substrate documentation](https://docs.substrate.io/reference/how-to-guides/parachains/add-hrmp-channels/) but applied to SORA.
```

File: src/establishing-hrmp-channel.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+Remember that HRMP channels usually go in one direction. For two-way communication, you'll need to set up another channel going the opposite way. Therefore, the following is split in to sub-sections:
+
+## Your Parachain to SORA
+
+1. Obtain the encoded call data of the `hrmp.hrmpInitOpenChannel` extrinsic on the relay chain.
+
+This special code includes a `recipient` detail that tells relay chain which parachain you would like to connect to. This detail changes depending on what environment you are planning to establish the channel in. For Kusama, look at [this list](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fksm-rpc.stakeworld.io#/parachains), and for Rococo, go to [this list](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frococo-rpc.polkadot.io#/parachains).
```
Comment (WRRicht3r) [inline src/establishing-hrmp-channel.md]:
```suggestion
This unique code includes a `recipient` detail telling the relay chain which parachain you want to connect to. This detail changes depending on what environment you are planning to establish the channel in. For Kusama, look at [this list](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fksm-rpc.stakeworld.io#/parachains), and for Rococo, go to [this list](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frococo-rpc.polkadot.io#/parachains).
```

File: src/establishing-hrmp-channel.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+::: info
+
+Combine this parachain id with the other info from the documentation, and you'll end up with something like this for Rococo: `0x3c00db070000e803000000900100`.
+
+:::
+
+2. Within your parachain, compose an XCM message that notifies the relay chain about the intent to open a channel with SORA. Refer to the official documentation for an illustrative example of the XCM message.
```
Comment (WRRicht3r) [inline src/establishing-hrmp-channel.md]: Please add a link to the docs being referred to here

File: src/establishing-hrmp-channel.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+::: info
+
+It's important to acknowledge that if your implementation of HRMP channel logic differs from the standard `polkadotXcm.send` pallet, you should modify this step accordingly.
```
Comment (WRRicht3r) [inline src/establishing-hrmp-channel.md]:
```suggestion
It is important to acknowledge that if your implementation of HRMP channel logic differs from the standard `polkadotXcm.send` pallet; you should modify this step accordingly.
```

File: src/establishing-hrmp-channel.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+:::
+
+Once the extrinsic executes successfully, SORA needs to accept the channel establishment.
+
+3. Subsequently, the SORA team will accept the channel by incorporating the encoded call data of `hrmp.hrmpAcceptOpenChannel`. This data will feature your parachain's ID as a parameter and will be executed via the `xcmApp.sudoSendXCM` call.
+
+Upon successful execution, the HRMP establishment from your parachain to SORA is finalized.
```
Comment (WRRicht3r) [inline src/establishing-hrmp-channel.md]:
```suggestion
Upon successful execution, (an) HRMP channel(s) will be established from your parachain to SORA.
```

**Review by WRRicht3r — CHANGES_REQUESTED** [review]

File: src/establishing-hrmp-channel.md (hunk 3 above)
Comment (bgodlin) [reply src/establishing-hrmp-channel.md]: Added in https://github.com/sora-xor/sora-docs/pull/136/commits/b24258351f26b32b522d6f87da90a59cf7a45743

**Review by WRRicht3r — APPROVED** [review]

**Pull request merged**